stable/namerd/templates/config.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  name: {{ template "fullname" . }}-config
+data:
+  config.yaml: |-
+    admin:
+      ip: 127.0.0.1
+      port: 9991
```

Comment:
This is namerd's admin port.  linkerd uses 9990 by default.  You can have the ip simply be `0.0.0.0` to expose it on non-local interfaces.